.github/workflows/static-content.yaml

```diff
@@ -0,0 +1,132 @@
+name: Static Content CI
+
+on:
+  pull_request:
+    branches: [ master ]
+    paths:
+      - '.github/workflows/**'
+      - 'content/**'
+      - 'client/flutter/assets/content_bundles/**'
+      - 'LICENSE'
+      - 'tools/**'
+  push:
+    paths:
+      - '.github/workflows/**'
+      - 'content/**'
+      - 'client/flutter/assets/content_bundles/**'
+      - 'LICENSE'
+      - 'tools/**'
+
+jobs:
+  build_content_bundles:
+    name: Build Content Bundles
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm install
+        working-directory: tools
+      - run: DONT_PUSH=true ./_deploy-static-serving.sh
```

Comment:
out of curiosity, why the leading `_` on the script filename?

Reply:
originally, it was a 'implementation' script, but now that i'm using it here, i'll remove the underscroe